frontend/src/components/AccountReceiveTable.vue

```diff
@@ -387,15 +400,37 @@ function useReceivedFundsTable(announcements: UserAnnouncement[], spendingKeyPai
   const isFeeLoading = ref(false);
   const isWithdrawInProgress = ref(false);
   const txHashIfEth = ref(''); // if withdrawing native token, show the transaction hash (if token, we have a relayer tx ID)
+  const vm = getCurrentInstance()!;
```

Comment:
_(arbitrary comment location to start a thread)_ 

There are lots of build errors, and `yarn build` fails to build the frontend

Reply:
Fixed with https://github.com/ScopeLift/umbra-protocol/pull/331/commits/2873e572201964a06f29866e835a418821b5bbd5

Reply:
Changed to `.tc` instead of `.toString()` in https://github.com/ScopeLift/umbra-protocol/pull/331/commits/4a5a09340b42c4e5eabcaf4ddb77beb6f7ce4ccd